BuildingBlocks/src/UnitTestTools/Data/TestDataGenerator.cs

```diff
@@ -14,7 +14,7 @@ public static string GenerateString(int resultLength, char[]? chars = null)
 
     public static IdentityAddress CreateRandomIdentityAddress()
     {
-        return IdentityAddress.Create(CreateRandomBytes(), "id1");
+        return IdentityAddress.Create(CreateRandomBytes(), "id1", "url");
```

Comment:
I would make more realistic examples:

```suggestion
        return IdentityAddress.Create(CreateRandomBytes(), "id1", "prod.enmeshed.eu");
```